src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public interface Figure {
+
+    double getArea();
+
+    void draw();
+}
```

Comment:
Don't repeat 'draw()' here. It is already in Drawable interface
```suggestion
public interface AreaCalculator {
    double getArea();
}
```